### DISCUSSION TITLE
doc(Search): No search results found for 'Select' component

### DISCUSSION THREAD

**Discussion by h2ls** [dotnetcore/BootstrapBlazor#3586] [Q&A] [answered]
## Issue Description

When using the documentation search functionality to find information on the `Select` component, no relevant search results are returned.

### Expected Behavior

Typing 'Select' into the search bar should yield documentation pages, usage examples, and API details for the `Select` component.

### Current Behavior

Currently, when entering 'Select' into the search bar, either no results are shown, or the results are not related to the `Select` component, leading to an inability to find the necessary documentation.

**Comment by bb-auto** [comment]
@azlis Thank you for submitting the document which is very important for our project.

**Comment by ArgoZhang** [comment, marked as answer]
@azlis 

![image](https://github.com/dotnetcore/BootstrapBlazor/assets/22001478/12c12e6d-3731-45b0-bd6f-aba27015be4a)

rebuild index
